src/app/modules/project/overview/overview.component.scss

```diff
@@ -267,7 +308,7 @@
 
     .card-body {
       margin-top: 10px;
-      background: #e9e9e9;
+      background: #F4F6F8;
```

Comment:
You can use the following variable here: $light-mode-background-color
Check /src/assets/styles/_variables.scss

We should try to use variables where we can so it's easier to make consistent theme changes across the platform.

Reply:
Yes sorry I couldn't find the file